dali/test/python/test_dltensor_operator.py

```diff
@@ -120,9 +137,9 @@ def mxnet_wrapper(fun):
     return lambda in1, in2: mxnet_adapter(fun, in1, in2)
 
 
-def mxnet_case(fun):
-    load_pipe = LoadingPipeline()
-    op_pipe = DLTensorOpPipeline(function=mxnet_wrapper(fun))
+def mxnet_case(fun, device):
```

Comment:
Maybe some common part of torch_case and mxnet_case can be extracted?

Reply:
I've extracted the common part from all of the test cases.